### PR TITLE
add more detail for measurement.

### PR DESCRIPTION
**Pull request by jyao1**
Signed-off-by: Jiewen Yao <jiewen.yao@intel.com>

### CODE REVIEW

File: source/5_security.rst [hunk 1]
```diff
@@ -381,3 +412,8 @@ At runtime, the system firmware may use a standard way - `SPDM <https://www.dmtf
 
 A platform should have a way to report a list of manifest and collect the runtime firmware measurement. As such, we can know the detailed firmware component information on a given platform.
 
+To supply the supply chain, the platform vendor should follow
```
Comment (paulmenzel) [inline source/5_security.rst:415]: “supply the supply chain” sounds strange. Maybe *provide*?

File: source/5_security.rst [hunk 2]
```diff
@@ -98,22 +98,53 @@ attack – recovery to an old known bad image.
 Measurment and Attestation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Trusted Boot (measured boot)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+OEM Component Measurement
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The platform shall enable trusted boot. The early boot component shall
+The platform shall follow `TCG PC Client Platform Firmware Profile <https://trustedcomputinggroup.org/resource/pc-client-specific-platform-firmware-profile-specification/>'_
```
Comment (paulmenzel) [inline source/5_security.rst:104]: What is the consequence? Is a TPM chip required now?

File: source/5_security.rst [hunk 3]
```diff
@@ -98,22 +98,53 @@ attack – recovery to an old known bad image.
...
+        * - Component
+          - PCR
+          - Event Type
+          - Event Data
+          - Event Log
+        * - UniversalPayload binary
+          - 0 (if it is part of OEM binary) or 2 (if it is treated as second phase loader)
+          - EV_EFI_PLATFORM_FIRMWARE_BLOB2
+          - UniversalPayload binary
+          - EFI_PLATFORM_FIRMWARE_BLOB2 structure (decriptor: "UniversalPayload")
```
Comment (paulmenzel) [inline source/5_security.rst:145]: de*s*criptor

File: source/5_security.rst [hunk 4]
```diff
@@ -98,22 +98,53 @@ attack – recovery to an old known bad image.
...
+
+
```
Comment (paulmenzel) [inline source/5_security.rst:147]: One blank line should be enough?

**Review by paulmenzel — COMMENTED** [review]
Please [remove the dot/period from the git commit message summary](https://cbea.ms/git-commit/).

File: source/5_security.rst (hunk 1 above)
Comment (jyao1) [reply source/5_security.rst:415]: yes, typo.

File: source/5_security.rst (hunk 3 above)
Comment (jyao1) [reply source/5_security.rst]: typo, fixed.

File: source/5_security.rst (hunk 4 above)
Comment (jyao1) [reply source/5_security.rst:146]: yes, fixed.

File: source/5_security.rst (hunk 2 above)
Comment (jyao1) [reply source/5_security.rst:104]: > Please [remove the dot/period from the git commit message summary](https://cbea.ms/git-commit/).

Done.

**Comment by jyao1** [comment]
> Please remove the dot/period from the git commit message summary

Done.

**Pull request merged**